teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Teacher
+  def initialize(specialization, age, name = 'unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
```

Comment:
- According to the project requirements, please ensure that your `Teacher` class inherits from the `Person` class. This [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance) will help you understand better. Please fix this 🙏